ci-operator/step-registry/openshift/e2e/aws/aws-load-balancer/openshift-e2e-aws-aws-load-balancer-commands.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+cat <<EOF | oc apply -f -
```

Comment:
I believe the repo is available when this script is run. So you should be able to create this from the repository directly. Better still read the relative path from an environment variable. Do not put the entire file in there. Adding more permissions later will be cumbersome.

Reply:
I've been trying to find out if the repository is actually available when this script is run and I'm not sure if it is. If it's not then refactor it to use the same environment variables for the namespace as the one used by the operator-subscribe.

Reply:
Couldn't find any docs regarding this. Couldn't find any other steps doing this either. According to the CI docs, the repo is cloned as layers ([as src](https://docs.ci.openshift.org/docs/architecture/ci-operator/#building-artifacts)) and doesn't really mention if steps (pre and post) are executed in the same layers (doesn't allow adding `from: src` tag either).

Reply:
>  refactor it to use the same environment variables for the namespace as the one used by the operator-subscribe.
We can't reuse `OO_INSTALL_NAMESPACE` since the namespace won't exist during this step (will be created later on). New variables can be introduced, though, if needed.

Reply:
We can reuse `OO_PACKAGE` and `OO_INSTALL_NAMESPACE` for the secret ref and the name of the `credentialsrequest`. Is that what you meant?

Reply:
This [step](https://github.com/openshift/release/blob/10549abd27aa8738f17d56a75576be4329e54f22/ci-operator/step-registry/baremetalds/metallb-e2e/test/baremetalds-metallb-e2e-test-ref.yaml#L3) uses `from: src`. 
> We can't reuse OO_INSTALL_NAMESPACE since the namespace won't exist during this step (will be created later on). New variables can be introduced, though, if needed.
Yes, parameterize the targets of CredentialsRequest.